parse/TechsParser.cpp

```diff
@@ -116,14 +116,14 @@ namespace {
 
             prerequisites
                 =   parse::label(Prerequisites_token)
-                >  (    '[' > +tok.string [ insert(_r1, _1) ] > ']'
+                >  (    ('[' > +tok.string [ insert(_r1, _1) ] > ']')
```

Comment:
the start of each 'line' of parser code should match up with the start of the other lines at the same level of nesting, as though the ( ) weren't there. so
````
                >  (    '[' > +tok.string [ insert(_r1, _1) ] > ']'
                    |   tok.string [ insert(_r1, _1) ]
                   )
````
could become
````
                >  (   ('[' > +tok.string [ insert(_r1, _1) ] > ']')
                    |   tok.string [ insert(_r1, _1) ]
                   )
````